cmd/check-gh-automation/README.md

```diff
@@ -1,5 +1,6 @@
 # Check GH Automation
 A tool to check that our bots (`openshift-merge-robot` and `openshift-ci-robot`) have access to repositories that have CI configured.
+It also checks that the app used to run it is installed in the repositories.
```

Comment:
English is so hard to me: Should we use `if` instead of `that` here?

Reply:
I think they are both okay. They both sound natural to me, and I found: https://forum.wordreference.com/threads/check-if-check-that.506018/